lib/elixir/lib/system.ex

```diff
@@ -1179,6 +1183,11 @@ defmodule System do
   defp cmd_opts([{:stderr_to_stdout, false} | t], opts, into, line),
     do: cmd_opts(t, opts, into, line)
 
+  defp cmd_opts([{:use_stdio, false} | t], opts, into, line) do
+    opts = opts -- [:use_stdio, :stderr_to_stdout]
```

Comment:
We cannot remove `stderr_to_stdout` after because there is no guarantee it was set before. For example, someone can do `use_stdio: false, stderr_to_stdout: true` and now it won't be removed. We should also support `use_stdio: true` and we can add a test for that.

Reply:
I've come up with something. However not very happy with the tests - they're on unix only for one - but obviously detaching the IO means we don't capture it anymore so it logs a bit on the console.

Reply:
(note about the `use_stdio: false, stderr_to_stdout: true` case, we could just forward that to erlang and it'll through, even though I added an extra check at the end for that to log our own message)